### DISCUSSION TITLE
"Begrenzung 0%" und "!" bei Einspeisung?

### DISCUSSION THREAD

**Discussion by GrimmiMeloni** [solectrus/solectrus#2871] [Q&A] [answered]
Siehe Betreff, sehe gerade folgendes:

<img width="335" alt="Bildschirmfoto 2024-02-29 um 14 58 06" src="https://github.com/solectrus/solectrus/assets/5851977/b9a33c85-26e8-4d13-bb5d-e76e43596fa7">

Habe ich vorher (letzten Sommer....) noch nie gesehen.

**Comment by ledermann** [comment, marked as answer]
Ja, passiert seit v0.14.2, Bug ist bereits behoben, Korrektur fließt ins nächste Update mit ein:
https://github.com/solectrus/solectrus/issues/2841
